src/Inputs/scene.inputManager.ts

```diff
@@ -823,7 +823,7 @@ export class InputManager {
             // Keyboard Events
             if (deviceType === DeviceType.Keyboard) {
                 evt.type = ((currentState === 1) ? "keydown" : "keyup");
-                evt.key = String.fromCharCode(inputIndex);
+                evt.key = eventData.key;

```

Comment:
Since we support IE you can fall back to String.fromCharCode in case .key is not defined

Reply:
I'll add code to cover for that scenario.  Great catch!